.eslintignore

```diff
@@ -1 +1,3 @@
 coverage
+node_modules
+support
```

Comment:
node_module should probably be ignored, but I think support should still be eslinted.